transient_model.py

```diff
@@ -0,0 +1,47 @@
+import numpy as np
+from build_matrix_A import build_matrix_A
+from build_matrix_b import build_matrix_b
+from matrix_solver import solve_matrix
+from build_A_analytical import build_A_analytical
+from generate_mesh import generate_mesh
+
+def transient_model(time, material_property_library, mesh, g_dot_list, boundary_conditions_list):
+    "Will also have to pass in however bcs or heat generation rate change with time"
+    "Begin by setting up the initial conditions"
+    """This function will calculate the transient response of the fuel particle to changes in either heat
+    generation rate or to boundary wall temperature. The A matrix has 1 added to its diagonal entries while
+    the b matrix has the previous temperature set added to its entries.

```

Comment:
I recommend some mathematical representation of what's happening here - why are A and b different as described?